### PR TITLE
Add a RUNPATH to installed libumf.so

### PR DESCRIPTION
**Pull request by PatKamin**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -312,6 +312,8 @@ endif()
 # dependencies.
 add_library(umf_headers INTERFACE)
 
+set(CMAKE_INSTALL_RPATH "\$ORIGIN")
```
Comment (lukaszstolarczuk) [inline CMakeLists.txt:315]: I think it requires at least a small-ish comment here 😉

Comment (bratpiorka) [inline CMakeLists.txt:315]: please move this to "install(FILES .. " with other installation-related code

Comment (PatKamin) [reply CMakeLists.txt]: Added a comment

Comment (PatKamin) [reply CMakeLists.txt]: Can't move it there as this variable has to be set before creating a umf target. Moved it to the CMakeLists file right before defining of a umf target.

**Review by bratpiorka — APPROVED** [review]

**Review by lukaszstolarczuk — APPROVED** [review]

**Pull request merged**